org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/ConnectionStreamFactory.java

```diff
@@ -153,6 +200,11 @@ private StreamProvider createProvider() {
 		if (port != null) {
 			return new SocketStreamProvider(host, Integer.parseInt(port));
 		}
+		final String lsHost = Environment.get("LS_SERVER_HOST", "localhost");
```

Comment:
use SERVER_HOST/SERVER_PORT to be consistent with CLIENT_* values

Reply:
SERVER_PORT is used in https://github.com/redhat-developer/vscode-java/blob/master/.vscode/launch.json#L26

Reply:
Fair enough. So in that case, use JDTLS_SERVER_*, so there's no ambiguity